packages/firestore/.eslintrc.js

```diff
@@ -27,6 +27,21 @@ module.exports = {
   rules: {
     'no-console': ['error', { allow: ['warn', 'error'] }],
     'import/no-default-export': 'error',
+    'import/order': [
+      'error',
+      {
+        'groups': [
+          'builtin',
+          'external',
+          'internal',
+          'parent',
+          'sibling',
+          'index'
+        ],
+        'newlines-between': 'always',
+        'alphabetize': { 'order': 'asc', 'caseInsensitive': false }
```

Comment:
My preference would be to make our import order case insensitive.

Reply:
Done.